### PR TITLE
Code style cleanup

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**
Resolves: https://github.com/blackbaud/skyux-core/issues/39

### CODE REVIEW

File: src/app/public/modules/format/app-format.spec.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// #region imports
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/format/app-format.spec.ts]: @Blackbaud-SteveBrush While I love collapsible regions, I noticed that VS Code will collapse imports automatically. Is this just being thorough?

File: src/app/public/modules/media-query/media-query-listener.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
+// #region imports
 import {
   SkyMediaBreakpoints
 } from './media-breakpoints';
+// #endregion
 
-export interface SkyMediaQueryListener {
-  (args: SkyMediaBreakpoints): void;
-}
+export type SkyMediaQueryListener = (args: SkyMediaBreakpoints) => void;
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/media-query/media-query-listener.ts]: Why are we switching this to a `type`?

File: src/app/public/modules/media-query/media-query.service.ts [hunk 3]
```diff
@@ -42,7 +44,7 @@ export class SkyMediaQueryService {
   private mdListener: MediaQueryListListener;
   private lgListener: MediaQueryListListener;
 
-  private currentSubject = new BehaviorSubject<SkyMediaBreakpoints>(this.current);
+  private _current = SkyMediaBreakpoints.md;
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/media-query/media-query.service.ts]: Should `_current ` and `currentSubject `be typed?

Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/media-query/media-query.service.ts]: In this case, the type is implied. This is a common pattern that reduces obvious redundancies:
```
private foo: string = 5;
```

It IS useful to declare a type on an instance variable when it defaults to `undefined`, however.

File: src/app/public/modules/format/app-format.spec.ts (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/format/app-format.spec.ts]: Huh, that must be a recent feature to VSCode. We should remove this requirement from the style guide, too.

File: src/app/public/modules/media-query/media-query-listener.ts (hunk 2 above)
Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/media-query/media-query-listener.ts]: screen shot 2018-12-13 at 1 05 08 pm

Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/media-query/media-query-listener.ts]: I get that linting error with our more strict tslint.json file. Shouldn't affect the consumer.

**Pull request merged**